project-base/storefront/components/Blocks/Product/ProductExposedStoresCount.tsx

```diff
@@ -20,7 +18,7 @@ export const ProductExposedStoresCount: FC<ProductExposedStoresCountProps> = ({
     }
 
     return (
-        <div className={twMergeCustom('text-sm text-black', className)} data-testid={TEST_IDENTIFIER}>
+        <div className="text-sm text-black" data-testid={TEST_IDENTIFIER}>
```

Comment:
Why was this removed? I see the classname being passed to this component.

Reply:
Hmm, probably I haven't noticed this one place and it was removed during my refactoring of ProductListItem since there was nothing left to be passed after refactor. Reverted back.